start.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# Start script for company-exemptions-data-api
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
```

Comment:
The app should be terminated if MESOS_SLAVE_PID isn't set.

This was previously needed by the deprecated Vagrant CHS development environment to source environment vars; our current dev environment requires environment variables to be specified per-application in a Docker compose file.